ProcessesApi.Tests/V1/Services/ChangeOfNameServiceTests.cs

```diff
@@ -76,6 +78,8 @@ public void ThrowsFormDataNotFoundException(string initialState, string trigger,
         [InlineData(SharedStates.DocumentsRequestedDes)]
         [InlineData(SharedStates.DocumentsRequestedAppointment)]
         [InlineData(SharedStates.DocumentsAppointmentRescheduled)]
+        [InlineData(SharedStates.TenureAppointmentRescheduled)]
```

Comment:
I don't think you can close a process from this state, just cancel? (looking at the spreadsheet)